packages/hebao_v1/contracts/stores/SecurityStore.sol

```diff
@@ -182,4 +289,30 @@ contract SecurityStore is DataStore
         wallets[wallet].inheritor = who;
         wallets[wallet].lastActive = uint128(now);
     }
-}
\ No newline at end of file
+
+    function cleanRemovedGuardians(address wallet)
```

Comment:
Can add a method or add some codes in this function and named function as CleanUpGuardians and also add effective guardian to guardians array.